api/all/src/main/java/io/opentelemetry/api/common/AttributeType.java

```diff
@@ -17,5 +17,32 @@ public enum AttributeType {
   STRING_ARRAY,
   BOOLEAN_ARRAY,
   LONG_ARRAY,
-  DOUBLE_ARRAY
+  DOUBLE_ARRAY;
+
+  /**
+   * Returns whether {@code this} is a primitive type or not.
+   *
+   * @return {@code true} if primitive, otherwise {@code false}
+   */
+  public boolean isPrimitive() {
```

Comment:
This could be implemented as an `EnumSet#contains`, it would save some lines and could be slightly more efficient